### PR TITLE
Added recipe for python package: periodictable

### PR DESCRIPTION
**Pull request by bjodah**
Let's see if this builds alright.

### CODE REVIEW

File: recipes/python-periodictable/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+{% set name = "python-periodictable" %}
```
Comment (synapticarbors) [inline recipes/python-periodictable/meta.yaml]: name should be 'periodictable' not 'python-periodictable'

File: recipes/python-periodictable/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```
Comment (synapticarbors) [inline recipes/python-periodictable/meta.yaml]: Looking at the setup.py, it looks like you need `numpy` and `pyparsing` as run requirements

File: recipes/python-periodictable/meta.yaml (hunk 1 above)
Comment (bjodah) [inline recipes/python-periodictable/meta.yaml]: does that go for the name of the folder as well? (I suspect that there may be other packages for other languages that share this name)

File: recipes/python-periodictable/meta.yaml (hunk 2 above)
Comment (bjodah) [inline recipes/python-periodictable/meta.yaml]: Sorry, that was an oversight. Will fix.

File: recipes/python-periodictable/meta.yaml (hunk 1 above)
Comment (synapticarbors) [inline recipes/python-periodictable/meta.yaml]: I think historically there had been discussion about making package names for python begin with `python-`, but it was abandoned. Under almost all circumstances the name matches the pypi package name.

Comment (bjodah) [inline recipes/python-periodictable/meta.yaml]: I see, do you want me to rename the folder then?

File: recipes/periodictable/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy
+    - pyparsing
+
+test:
+  # Some package might need a `test/commands` key to check CLI.
```
Comment (jakirkham) [inline recipes/periodictable/meta.yaml]: Please drop this comment.

File: recipes/python-periodictable/meta.yaml (hunk 1 above)
Comment (jakirkham) [inline recipes/python-periodictable/meta.yaml]: Yes, the directory should match the package name. It's a bit silly, but we have issues if it doesn't.

xref: https://github.com/conda-forge/staged-recipes/issues/1073

**Comment by bjodah** [comment]
Same thing here, the appveyor build is complete but shown as waiting..

**Comment by jakirkham** [comment]
Yeah, sometimes the webhook fails to update the status. Restarting AppVeyor is the only fix I know of. Would do that normally, but the queue is a bit backed up ATM so will not now.

**Comment by jakirkham** [comment]
Thanks @bjodah for the contribution and @synapticarbors for reviewing.

**Comment by bjodah** [comment]
Thanks @jakirkham and @synapticarbors for the review!

**Pull request merged**